code/game/mecha/mech_sensor.dm

```diff
@@ -13,77 +13,4 @@
 	var/id_tag = null
 
 	var/frequency = 1379
-	var/datum/radio_frequency/radio_connection
```

Comment:
Don't be deleting mechas, this is unnecessary. We will probably find a use in future for their code, it is very flexible.

Disable them ingame, make sure parts and fabricators can't be made, but don't delete things